db/migrate/20230117072559_create_like.rb

```diff
@@ -0,0 +1,9 @@
+class CreateLike < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
```

Comment:
Great job @Mhamad-Raad 🤝

- [ ] [Optional] You may add `t.references :author, foreign_key: { to_table: 'users' }` to the likes, comments, and posts tables.